Sources/VLCSettingsTableViewCell.swift

```diff
@@ -30,12 +30,10 @@ class VLCSettingsTableViewCell: UITableViewCell {
             let toggle = IASKSwitch(frame: .zero)
             toggle.addTarget(target, action: #selector(target.toggledValue(_:)), for: .valueChanged)
             accessoryView = toggle
-            selectionStyle = .none
-        case kIASKOpenURLSpecifier, kIASKPSMultiValueSpecifier:
+        case kIASKOpenURLSpecifier:
             accessoryType = .disclosureIndicator
-            selectionStyle = .default
         default:
-            assertionFailure("\(reuseIdentifier) has not been defined for VLCSettingsTableViewCell")
+            accessoryType = .none
```

Comment:
Can we handle it explicitly for the known cases and keep the assertionFailure?

Reply:
Yes! Don't know what I was thinking :p